cli/src/helpers/installPackages.ts

```diff
@@ -6,9 +6,9 @@ import {
 } from "~/installers/index.js";
 import { logger } from "~/utils/logger.js";
 
-type InstallPackagesOptions = {
+type InstallPackagesOptions = InstallerOptions & {
   packages: PkgInstallerMap;
-} & InstallerOptions;
+};
```

Comment:
https://v6--typescript-eslint.netlify.app/rules/sort-type-constituents - is newly in the _`stylistic`_ config. While it does have an autofixer, I do worry that it's too nitpicky and will annoy people. Thoughts?

Reply:
i think i like it, makes it more like interface where the things you extend comes first

Reply:
Great - if we end up removing it from the config, I can manually re-anble it in this PR.